Source/WTF/wtf/HashTable.h

```diff
@@ -1558,7 +1558,7 @@ DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(HashTable);
         const ValueType* operator->() const { return get(); }
 
         HashTableConstIteratorAdapter& operator++() { ++m_impl; return *this; }
-        // postfix ++ intentionally omitted
+        HashTableConstIteratorAdapter& operator++(int) { auto prev = *this; ++m_impl; return prev; }
```

Comment:
WebKit coding style discourages the use of abbreviations like `prev` and encourages use of full words instead.